File: abc/abc174/abc174_d.cs
// https://atcoder.jp/contests/abc174/submissions/15666603

using System;

class Program
{
    static void Main()
    {
        var N = int.Parse(Console.ReadLine());
        var S = Console.ReadLine();

        var l = 0;
        var r = N - 1;
        var result = 0;

        while (true)
        {
            if (r <= l)
            {
                break;
            }
            if (S[l] == 'R')
            {
                l++;
            }
            else if (S[r] == 'W')
            {
                r--;
            }
            else
            {
                l++;
                r--;
                result++;
            }
        }
        Console.WriteLine(result);
    }
}

File: abc/abc174/abc174_d.fs
// https://atcoder.jp/contests/abc174/submissions/15666387

let N = stdin.ReadLine() |> int
let S = stdin.ReadLine()

let rec solver l r i =
    if r <= l then i
    elif S.[l] = 'R' then solver (l + 1) r i
    elif S.[r] = 'W' then solver l (r - 1) i
    else solver (l + 1) (r - 1) (i + 1)

solver 0 (S.Length - 1) 0 |> stdout.WriteLine
